### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by jenjenp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,34 @@
...
+
+## List of UDF libraries in this repository:
+
+### Anomaly Functions
+#### 1. Relative Threshold -
+Compare the event volume within the view against a target threshold and rate. The rate is defined as number of events / throughput. Throughput can set as the number of times the method containing the event was called, or the number of times the application thread calling into the event executed.
+See code at: https://git.io/fx6sl
+#### 2. Automatic Entry point Timers -
+Automatically set timers on application entry points based on average runtime.
+See code at: https://git.io/
```
Comment (chook) [inline README.md]: no code?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,34 @@
 # overops-functions
 Public OverOps User Defined Functions
+
+Full manifest of all libraries can be found here - https://git.io/fxDIW
+
+## Our Collection consists of two types of functions:
+### Anomaly Functions:
+An Anomaly UDF is a function that would be used to determine whether the volume of specific events is considered anomalous by comparing it to a target threshold within a view/timeframe.
```
Comment (chook) [inline README.md]: i would add that this is a periodic check and that it can be adjusted to 5/15/60... minutes or something

File: README.md [hunk 3]
```diff
@@ -1,2 +1,34 @@
...
+#### 2. Automatic Entry point Timers -
+Automatically set timers on application entry points based on average runtime.
+See code at: https://git.io/
+#### 3. Severity -
+Mark events as New if they were introduced in an active deployment and are important. Mark events as Regressed if they have crossed a volume threshold against a previous timeframe.
+See code at: https://git.io/fxiSl
+
+### Channel Functions:
+#### 1. Routing -
+Classify incoming events according to the functional component within the code from which they originated.
```
Comment (chook) [inline README.md]: as some examples and how the label is generated.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,34 @@
 # overops-functions
 Public OverOps User Defined Functions
+
+Full manifest of all libraries can be found here - https://git.io/fxDIW
+
+## Our Collection consists of two types of functions:
```
Comment (chook) [inline README.md]: perhaps we should also mention the new "install" function that runs when the UDF is saved

File: README.md (hunk 1 above)
Comment (jenjenp) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (jenjenp) [reply README.md]: added

File: README.md (hunk 3 above)
Comment (jenjenp) [reply README.md]: clarified + added examples

**Pull request merged**